classes/userPrivateNote/UserPrivateNote.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/**
+ * @file classes/user/UserPrivateNote.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class UserPrivateNote
+ *
+ * @ingroup user
+ *
+ * @see DAO
+ *
+ * @brief Basic class describing user private note existing in the system.
+ */
+
+namespace PKP\userPrivateNote;
+
+use PKP\core\DataObject;
+
+class UserPrivateNote extends DataObject
```

Comment:
No need to change this unless you're motivated to -- but in general we're moving towards Eloquent models rather than `DataObject`s. At some point in the distant future we'll deprecate `DataObject` entirely.

Reply:
...but [this](https://github.com/pkp/pkp-lib/pull/9549#discussion_r1419499935) would be my preference rather than moving to Eloquent.